SupersonicSound/Wrapper/Util.cs

```diff
@@ -207,5 +208,14 @@ public static Guid FromFmod(this GUID guid)
                 guid.Data4[7]
             );
         }
+
+        public static bool IsLinux
+        {
+            get
+            {
+                int p = (int)Environment.OSVersion.Platform;
```

Comment:
That is weird, but I guess it makes sense. Could you add a link to that SO answer in a comment on the code too please.